resources/teamcity/android/android-actions.inc.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Keyman is copyright (C) SIL Global. MIT License.
+
+android_clean_action() {
+  builder_echo start "clean" "Cleaning artifact directories"
+  # shellcheck disable=SC2154
+  "${KEYMAN_ROOT}/android/build.sh" clean
+  builder_echo end "clean" success "Finished cleaning artifact directories"
+}
+
+android_build_action() {
+  builder_echo start "build" "Building Keyman for Android"
```

Comment:
It seems unnecessary to have two lots of headers (here and in `builder_run_action`)?